From: Boyuan Yang <byang@debian.org>
Date: Mon, 18 Dec 2023 22:53:24 -0500
Subject: tox.ini: Use py3 versions

---
 tox.ini | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/tox.ini b/tox.ini
index 7c96eb3..906d768 100644
--- a/tox.ini
+++ b/tox.ini
@@ -44,17 +44,17 @@ commands =
 
     # this creates versioneer.py in the current directory, which is used by
     # tests
-    python setup.py make_versioneer
-    python -m unittest discover test
-    python test/git/test_git.py -v
-    python test/git/test_invocations.py -v
+    python3 setup.py make_versioneer
+    python3 -m unittest discover test
+    python3 test/git/test_git.py -v
+    python3 test/git/test_invocations.py -v
 
     # this creates git_version.py, which is used by tests
-    python setup.py make_long_version_py_git
-    pyflakes setup.py versioneer.py git_version.py
+    python3 setup.py make_long_version_py_git
+    pyflakes3 setup.py versioneer.py git_version.py
     # pyflakes all of src except src/git/long_header.py and src/header.py
-    python test/run_pyflakes_src.py
-    pyflakes test
+    python3 test/run_pyflakes_src.py
+    pyflakes3 test
     flake8 git_version.py versioneer.py
     pycodestyle --max-line-length=88 git_version.py versioneer.py
     !pypy3: mypy git_version.py versioneer.py src/installer.py setup.py
